Application/Commands/Channels/CreateChannel/CreateChannelRequestHandler.cs

```diff
@@ -1,33 +1,31 @@
 ﻿using Application.Interfaces;
 using Application.Models;
 using Application.Providers;
+using AutoMapper;
 using MediatR;
 
 namespace Application.Commands.Channels.CreateChannel
 {
-    public class CreateChannelRequestHandler : RequestHandlerBase,  IRequestHandler<CreateChannelRequest, Channel>
+    public class CreateChannelRequestHandler : RequestHandlerBase, IRequestHandler<CreateChannelRequest, Channel>
     {
-
         public async Task<Channel> Handle(CreateChannelRequest request, CancellationToken cancellationToken)
         {
             //TODO: Перевірити що у юзера є відповідні права
-            Server server = await Context.FindByIdAsync<Server>(request.ServerId, cancellationToken, "ServerProfile");
-            List<User> users = new();
-            server.ServerProfiles.ForEach(profile => users.Add(profile.User));
+            Server server = await Context.FindSqlByIdAsync<Server>(request.ServerId, cancellationToken, "ServerProfile");
```

Comment:
Неправильные импроты
```suggestion
            Server server = await Context.FindSqlByIdAsync<Server>(request.ServerId, cancellationToken, "ServerProfiles","ServerProfiles.User");
```

Reply:
Ні... User'a не треба підтягувати. Через include треба підтягувати тільки списки, бо з цим крашиться

Reply:
Без этого крашится, потому что в строчке 16 user - null

Reply:
Теперь работает